theano/tensor/signal/tests/test_downsample.py

```diff
@@ -193,12 +193,21 @@ def test_DownsampleFactorMax(self):
 
                 # DownsampleFactorMax op
                 maxpool_op = DownsampleFactorMax(maxpoolshp,
-                                                 ignore_border=ignore_border,
-                                                 mode=mode)(images)
+                                                ignore_border=ignore_border,
+                                                mode=mode)(images)
```

Comment:
Yeah, will do that. (Actually, i just ran this particular test alone, and since it failed, didn't run the entire test..)